tests/support/expectations.js

```diff
@@ -661,16 +664,6 @@ module.exports = function (request, config) {
       }
     },
     'Header setting': {
-      'setting mocked headers': function (test) {
```

Comment:
OK to keep only one test on setting headers but I would rather keep that one with `'https://authorized.example/'` to test the mocked way (and just modify line 670), WYT ? 🐤